mvp.feature

```diff
@@ -208,278 +208,277 @@ Feature: User with no Author identity in the Authority Register for Personas see
 
   # Menu items for logged in person
-  Scenario: User opens the page Min Profil
+  Scenario: 28. User opens the page Min Profil
     Given the user is logged in
     When they click the menu item Min Profil
     Then the page Min Profil is opened
-    # Write a new Scenario for this
+  # Write a new Scenario for this
 
-  Scenario: User logs out
+  Scenario: 29. User logs out
     Given the user is logged in
     When they click the menu item Logg ut
     Then the user is logged out from Feide
-	And the search page is opened
+    And the search page is opened
     And the user sees non-logged-in menu
 
-   # Menuitems for Registrator
-  Scenario: User opens Ny Registrering
+  # Menuitems for Registrator
+  Scenario: 30. User opens Ny Registrering
     Given the user is logged in as Registrator
     When they click the menu item Ny Registrering
     Then the page Ny Registrering is opened
-	And the user see tabs
-        | Start med å laste opp fil |
-        | Start med en lenke til publikasjon |
-        | Start med forslag fra din ORCID |
+    And the user see tabs
+      | Start med å laste opp fil          |
+      | Start med en lenke til publikasjon |
+      | Start med forslag fra din ORCID    |
 
-  Scenario: User opens Mine Publiseringer
+  Scenario: 31. User opens Mine Publiseringer
     Given the user is logged in as Registrator
     When they click the menu item Mine Registreringer
     Then the page Mine Registreringer is opened
-	And a list of all unpublished registrations with the fields
-        | Navn på publisering |
-        | <Status> |
-        | Opprettet |
-        Examples:
-            | Status |
-            | Kladd |
-            | Avvist |
+    And a list of all unpublished registrations with the fields
+      | Navn på publisering |
+      | <Status>            |
+      | Opprettet           |
+    Examples:
+      | Status |
+      | Kladd  |
+      | Avvist |
     And each list item has a button Slett and Rediger that is enabled
 
   # Actions from Page : Mine Publiseringer (Rediger)
-  Scenario: User opens an item in the Mine Publiseringer list
+  Scenario: 32. User opens an item in the Mine Publiseringer list
     Given the user is logged in as Registrator
-	And has opened the page Mine Publiseringer
+    And has opened the page Mine Publiseringer
     When they click Rediger on an item
     Then the item is opened in the wizard
-	And the user sees the Beskrivelse tab
+    And the user sees the Beskrivelse tab
     And the fields in the wizard are populated with values from the saved publication
 
   # Actions from Page : Mine Publiseringer (Slett)
-  Scenario: User opens an item in the Mine Publiseringer list
+  Scenario: 33. User opens an item in the Mine Publiseringer list
     Given the user is logged in as Registrator
-	And has opened the page Mine Publiseringer
+    And has opened the page Mine Publiseringer
     When they click Slett on an item
     Then a comfirmation pop-up is opened
```

Comment:
Dette scenariet bør kanskje splittes i to? Hva skjer når man trykker på Yes? Hvor havner brukeren?